cpusupport/cpusupport_arm_aes.c

```diff
@@ -31,6 +35,20 @@ CPUSUPPORT_FEATURE_DECL(arm, aes)
 	supported = (capabilities & HWCAP2_AES) ? 1 : 0;
 #endif
 #endif /* CPUSUPPORT_HWCAP_GETAUXVAL */
+
+#if defined(CPUSUPPORT_HWCAP_ELF_AUX_INFO)
+	unsigned long capabilities;
+
+#if defined(__aarch64__)
+	if (elf_aux_info(AT_HWCAP, &capabilities, sizeof(unsigned long)))
+		return (0);
+	supported = (capabilities & HWCAP_AES) ? 1 : 0;
+#else
+	if (elf_aux_info(AT_HWCAP2, &capabilities, sizeof(unsigned long)))
```

Comment:
This code (for non-64bit arm) is untested, but it's so similiar to the Linux arm32 code that I think it's worth including.